sections/oxygen_introduction.md

```diff
@@ -86,7 +86,7 @@ The RBRcoda T.ODO uses the same foils and methods as the 4831 and 4831F so every
 RBR refers to the standard optode (~30 s $\tau$) foil as “slow” and the fast (~8 s) as the standard.
 They also use further foil design (~1 s response) which they call fast. 
 The RBR sensor has a smaller form factor than the Aandera optodes, but is overall more similar to a 4831 with the temperature sensor very closely located to the sensing foil. 
-This sensor is still fairly recent on gliders and little is known for now.
+This sensor has recently been implemented in ocean glider, and little has known about their performance.
 
```

Comment:
There were also some grammar issues in the new sentence. I will reformulate. To:

`This sensor has recently been implemented in gliders, and little is known about their performance.`